content/en/about/feature-stages/index.md

```diff
@@ -16,17 +16,29 @@ within the project, not to the project as a whole. Here is a high level descript
+| `Upgradeability`      | The schema and semantics of an experimental feature may change in newer versions without any guarantees of preserving backwards compatibility requiring configuration changes during upgrades. API versions may increment faster than the monthly release cadence and developers are not required to maintain multiple versions for backwards compatibility. Developers are encouraged to increment the API version when schema or semantics change in an incompatible way | The schema and semantics of an alpha feature may change in a later software release, without any provision for preserving configuration objects in an existing Istio installation API versions may increment faster than the monthly release cadence and the developer need not maintain multiple versions Developers should increment the API version when schema or semantics change in an incompatible way                                      | The schema and semantics may change in a later software release When this happens, an upgrade path will be documented In some cases, objects will be automatically converted to the new version In other cases, a manual upgrade may be necessary A manual upgrade may require downtime for anything relying on the new feature, and may require manual conversion of objects to the new version When manual conversion is necessary, the project will provide documentation on the process | Only strictly compatible changes are allowed in subsequent software releases                                                               |
```

Comment:
I wouldn't use quotes around Upgradability as it becomes the only highlighted entry. I suspect it's because of a spelling issue. I would add it to .spelling if that is the case

Reply:
Also we mention a monthly cadence?

Reply:
Removed the notion of a monthly cadence. This was taken from https://github.com/istio/community/blob/master/FEATURE-LIFECYCLE.md

It obviously needs more work before we can merge this in.